### DISCUSSION TITLE
Paperparrot with cloudflare zero tunnels

### DISCUSSION THREAD

**Discussion by vednolacni** [LeoKlaus/Paperparrot#191] [Q&A]
Hello,

im looking for a instructions on how to get Paperparrot working over cloudlfare zero tunnels. 

If i want to log in it give me a Unexpected response:200 but its working from the web.

If anybody got something i would appreciate it, because im new to all this...

**Comment by LeoKlaus** [comment]
Hey vednolacni,

do you use cloudflare tunnels or cloudflare zero trust (additional authentication)?

Anything that blocks access to the /api/ path of your Paperless instance will not work with Paperparrot as of now.

If you host Paperless on a machine at home, something like TailScale might be a viable and secure alternative to Cloudflares zero trust.

**Comment by vednolacni** [comment]
I think I'm using zero trust.

Im hosting on my server yes but wanted to use cloudflare like for all other
things. So for now this is not an option?

V V pon., 19. feb. 2024 ob 12:52 je oseba Leo Wehrfritz <
***@***.***> napisala:

**Comment by LeoKlaus** [comment]
> I think I'm using zero trust. Im hosting on my server yes but wanted to use cloudflare like for all other things. 

Zero trust would mean you have to authenticate in some way or form before getting to the Paperless login screen (i.e. with GitHub, Google,...).
If that's the case, then there really is no way to make this work with Paperparrot as of now.

You can still send me the debug logs after attempting to sign in. You can get them by tapping the button in the top right of the login page in Paperparrot.
